generic/tests/dd_test.py

```diff
@@ -17,7 +18,7 @@
 try:
     from itertools import zip_longest as zip_longest
 except Exception:
-    from itertools import izip_longest as zip_longest
+    from itertools import zip_longest as zip_longest
```

Comment:
@luckyh on Python3 there is no `izip_longest` on `zip_longest` is available.

Reply:
I think this line is for py2, because:
```python
Python 2.7.15 (default, Oct 15 2018, 15:26:09) 
[GCC 8.2.1 20180801 (Red Hat 8.2.1-2)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from itertools import zip_longest as zip_longest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name zip_longest
>>> from itertools import izip_longest as zip_longest
>>> 
```
```python
Python 3.7.3 (default, Mar 27 2019, 13:41:07) 
[GCC 8.3.1 20190223 (Red Hat 8.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from itertools import zip_longest as zip_longest
>>>
```

Reply:
@luckyh yes, it might be that we need to add a condition to `izip_longest` in the case of `python2` and `zip_longest` in the case of `python3`.